_posts/2017-02-21-ome-files-c-0-3-1.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: Release of OME Files C++ 0.3.1
+intro-blurb: The OME team is pleased to announce the release of OME Files C++ 0.3.1
+---
+OME Files is a reference implementation of the OME Data Model and OME-TIFF file format for the storage and interchange of biological imaging data and metadata.  It provides support for:
+
+-  the OME Data Model (model objects, metadata store, XSL transforms and XML validation)
+-  OME-TIFF reading
+-  OME-TIFF writing
+
+This release contains two bugfixes to correct the support for references in the data model such as ROI references in an Image.
+
+**Download**
+
+The software release is available at 
+
+[archived downloads](http://downloads.openmicroscopy.org/ome-files-cpp/0.3.1/)
+
+This page includes links to the individual repositories and source releases, as well as the documentation and binary builds of the collection for several common platforms, including Windows, Linux and MacOS X.
+
+**Upgrading from 0.3.0**
+
+No changes required.
+
+**Changes since ome-files-cpp 0.3.0**
+
+*OME Common*
+
+No changes
+
+*OME Model*
+
+-  xsd-fu: C++ OMEXMLModelObject template inserts missing links
+
+*OME Files*
+
+-  MetadataTools: Resolve references after updating model objects
+-  Add pixel buffer interleaving example
+-  Update copyright dates
+
+*OME QtWidgets*
+
+No changes
+
+*Super-Build 0.3.1*
+
+Package changes:
+
+-  ome-files 0.3.1: Updated package
+-  ome-model 5.5.1: Updated package
+
+**Future changes**
+
+Upcoming improvements include:
+
+*All components*
+
+-  Enabling of additional C++11 features following testing of compiler support and compatibility
+-  Further performance profiling and scalability improvements
+
+*OME XML*
+
+-  Support for the use of custom annotations with the OME-XML data model API
+-  This generalized annotation support will also be used for existing annotations including Modulo and OriginalMetadata
+-  Improved and more comprehensive OME-XML and OME-TIFF samples
+-  Improved documentation and code samples
+
+*OME Files*
+
+-  Improvements to automated integration testing of OME-TIFF files
+
+*Super-Build*
+
+-  Further improvements for building on Windows, including initial VS2017 support
+-  Updating of the set of platforms binary builds are provided for
+
+You can find further information on which features will be included in upcoming releases via the Trello board 
+
```

Comment:
Don't need all these blank lines here because the new formatting is much more readable - I'd put the link and the additional text in brackets all in once sentence